app/views/issues/evidence/_table.html.erb

```diff
@@ -0,0 +1,53 @@
+<table class="table table-striped mb-0"
+  data-behavior="dradis-datatable"
+  data-default-columns='["Node", "Created by"]'
+  data-item-name=<%= "issue_#{@issue.id}_evidence" %>
+  data-local-storage-key="project.ce.<%= dom_id(current_project) %>.<%= "issue_#{@issue.id}_evidence_datatable" %>">
+  <thead>
+    <tr>
+      <th class="no-sort" data-column-visible="false"><span class="sr-only">Select</span></th>
+      <% @evidence_columns.each do |column| %>
+        <th><%= column %></th>
+      <% end %>
+      <th class="no-sort" data-column-visible="false"><span class="sr-only">Actions</span></th>
+    </tr>
+  </thead>
+  <tbody>
+    <% @affected_nodes.each do |node| %>
+      <% node.evidence.where(issue_id: @issue.id).each do |evidence| %>
+        <tr id="evidence-<%= evidence.id %>">
+          <td class="select-checkbox" data-behavior="select-checkbox"></td>
+          <% @evidence_columns.each do |column| %>
+            <%
+              sort, display =
+              case column
+              when 'Node'
+                [evidence.node.label, link_to(evidence.node.label, [current_project, node, evidence])]
```

Comment:
`Label` quacks better than this `Node` field

Reply:
I thought `Node` would be more clear to users when they are looking at the table but now that I'm looking at this again I have pushed a change to replace `Node` with `Affected` for consistency with `issues#index`